cmd/manager/main.go

```diff
@@ -20,8 +20,13 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/IBM/ibm-block-csi-operator/pkg/controller/ibmblockcsi/syncer"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 	"os"
 	"runtime"
+	"strings"
```

Comment:
Is there a code style rule for packages order?

Reply:
the auto one (ctrl alt o) put it there..

Reply:
@oriyarde Is it the best way at the moment to order the packages?

Reply:
currently yes